src/translib/transformer/xfmr_vrf.go

```diff
@@ -232,27 +267,39 @@ var network_instance_table_name_xfmr TableXfmrFunc = func (inParams XfmrParams)
         }
 
         /*
-         * for GET or DELETE, use the key "name" only to get the DB table name
-         * for CREATE, UPDATE or REPLACE, use the key "name" and type to get the DB table name
-         */
-        if ((inParams.oper == CREATE) ||
-            (inParams.oper == UPDATE) ||
-            (inParams.oper == REPLACE))  {
-                /* get the type for the network instance config */
+         * For CREATE or PATCH at top level (Network_instances), check the config type if user provides one 
+         * For other cases of UPATE, CREATE, or GET/DELETE, get the table name from the key only
+         */ 
+        if (((inParams.oper == CREATE) ||
+             (inParams.oper == REPLACE) ||
+             (inParams.oper == UPDATE)) &&
+             (inParams.requestUri == "/openconfig-network-instance:network-instances")) {
```

Comment:
UPDATE aka PATCH at the top network-instances level is a create based on Kwan, pl refer to rfc8040.